code/parse/parselo.h

```diff
@@ -158,8 +158,8 @@ extern void parse_string_flag_list(int *dest, flag_def_list defs[], size_t defs_
 // A templated version of parse_string_flag_list, to go along with the templated flag_def_list_new.
 // If the "is_special" flag is set, or a string was not found in the def list, it will be added to the unparsed_or_special_strings Vector
 // so that you can process it properly later
```

Comment:
Can you update the comment to account for the new parsing capability?

Also it would be good to generally explain what the `special_flag_def_list_new` type does and how it can be used.